### PR TITLE
python3 compiling cleanup (continued)

### PR DESCRIPTION
**Pull request by davidlange6**
various fixes for invalid python syntax. Main themes
1) change the ending of CondCore/Utilities/scripts/fnget.py as its a bash script
2) FWLiteEnabler::enable() is C not python. Replaced by ROOT.FWLiteEnabler.enable()
3) '\u' needs to be explicitly escaped as '\\u' in python3

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: HLTrigger/Tools/scripts/timingPdfMaker.py [hunk 1]
```diff
@@ -36,443 +36,443 @@
...
-                texfile.write('\\newpage \subsection{'+ path.replace('_','\_') +'} \centering \includegraphics[scale=0.6]{incPathTime'+ path.replace('_','') +'temp.png}\n')
-                get_plot1(infile,'incPathTime_'+path)
-        texfile.write('\end{document}')
-        texfile.close()
-
-        texfile.close()
-
-	get_plot1(infile,'totalTime')
+    texpreamble = ['\documentclass[10pt,a5paper,landscape]{report}\n',
+                   '\\usepackage{graphicx}\n',
```
Comment (Martin-Grunewald) [inline HLTrigger/Tools/scripts/timingPdfMaker.py]: I do not understand the changes here: this line starts with a double back-slash, but not the line above,
while both lines should produce LaTeX text lines starting with a single back-slash each.
??

**Comment by davidlange6** [comment]
as I explained briefly in the PR, \u has a special meaning. Eg, in python2

>>> d='\u'
>>> d
'\\u'

whereas in python3

>>> d='\u'
  File "<stdin>", line 1
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 0-1: truncated \uXXXX escape
>>> d='\\u'
>>> d
'\\u'

so I believe the behavior is unchanged. I tried a simple test and the output is the same for writing strings to text files if I use '\u....' or '\\u.....'.

**Comment by Martin-Grunewald** [comment]
+1

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by fabiocos** [comment]
@davidlange6 this PR needs to be rebased at this point

**Comment by davidlange6** [comment]
Your PR is unmergeable. Please have a look and possibly rebase it.

**Comment by davidlange6** [comment]
please test

**Comment by perrotta** [comment]
+1
- Just rebased since last reco signature

**Comment by Martin-Grunewald** [comment]
+1

**Comment by andrius-k** [comment]
+1

**Comment by ggovi** [comment]
+1

**Comment by kpedro88** [comment]
@davidlange6 a comment for the future: when writing TeX strings in python, I find it easier just to mark them all as raw strings like `r"\u..."`, rather than trying to escape [backslashes](https://xkcd.com/1638/) everywhere.

**Comment by kpedro88** [comment]
+1

**Comment by kpedro88** [comment]
merge

**Comment by kpedro88** [comment]
@davidlange6 it would be useful to backport at least the parts of this affecting `pileupCalc.py` to 10_2_X, which is now the recommended analysis release (otherwise the script is broken).

**Pull request merged**